lib/server/controllers/logs.controller.js

```diff
@@ -54,6 +58,11 @@ const createLog = async (request, response, next) => {
         response.status(Number(error.status)).json({ errors: [error] });
     } else {
         response.status(201).json({ data: result });
+        const url = `${request.protocol}://${request.get('host')}?page=log-detail&id=${result.id}`;
+        const tags = result.tags.map((tag) => tag.text);
+        if (notification && notification.isConfigured()) {
```

Comment:
Here you could use the same syntax you used above
```suggestion
        if (notification?.isConfigured()) {
```